sites/partners/cypress/integration/03-listing.spec.ts

```diff
@@ -0,0 +1,257 @@
+describe("Listing Management Tests", () => {
```

Comment:
If we want to bring these over, we'll have to modify them, at the very least the differences around units will have to be accounted for.